_tests/managed_tests/README.md

```diff
@@ -1,42 +1,176 @@
-## Quality Insights Toolkit self-tests
+# QIT Self-Tests
 
-This directory contains self-tests for the Quality Insights Toolkit.
+These self-tests validate that the **Quality Insights Toolkit (QIT)** flags expected issues for various WordPress environments and configurations.
 
-The self-tests uses Snapshot Testing to validate that the QIT flags the expected issues given a certain input.
+## Overview
 
-### Self-tests
-- Each directory is a test-type self-test, except for "vendor" and "tests" folders, eg: `activation`, `security`, `e2e`, etc.
-- Each test-type self-test can contain multiple tests, eg: `activation/test-1`, `activation/test-2`, etc.
-- Each test has a `env.php` file, with parameters used for the test, such as PHP, WooCommerce and WordPress versions, eg: `activation/test-1/env.php`.
+1. **Discovery & Packaging**
+  - Each **test type** (e.g., `activation`, `security`, `woo-e2e`) has one or more **test scenarios** (e.g., `test-1`, `scenario-a`).
+  - The script packages each scenario’s System Under Test (SUT) into a ZIP (unless reusing JSON snapshots).
 
-### Running the self-tests
+2. **Dispatch**
+  - The script uses `/home/lucas/automattic/qit-cli/qit` to **run** each scenario test asynchronously, collecting `test_run_id`s.
```

Comment:
I think we should remove the dir path here and below, so it's like this:

```suggestion
  - The script uses `qit-cli/qit` to **run** each scenario test asynchronously, collecting `test_run_id`s.
```

I might even reword this to something like:

```
  - The script uses the binary at `qit-cli/qit` to **run** each scenario test asynchronously, collecting `test_run_id`s.
```